common/locales/en/questsContent.json

```diff
@@ -69,7 +69,7 @@
   "questSpiderUnlockText": "Unlocks purchasable spider eggs in the Market",
 
   "questVice1Text": "Vice, Part 1: Free Yourself of the Dragon's Influence",
-  "questVice1Notes": "<p>They say there lies a terrible evil in the caverns of Mt. Habitica. A monster whose presence twists the wills of the strong heroes of the land, turning them towards bad habits and laziness! The beast is a grand dragon of immense power and comprised of the shadows themselves: Vice, the treacherous Shadow Wyrm. Brave Habiteers, stand up and defeat this foul beast once and for all, but only if you believe you can stand against its immense power. </p><h3>Vice Part 1: </h3><p>How can you expect to fight the beast if it already has control over you? Don't fall victim to laziness and vice! Work hard to fight against the dragon's dark influence and dispel his hold on you! </p>",
+  "questVice1Notes": "They say there lies a terrible evil in the caverns of Mt. Habitica. A monster whose presence twists the wills of the strong heroes of the land, turning them towards bad habits and laziness! The beast is a grand dragon of immense power and comprised of the shadows themselves: Vice, the treacherous Shadow Wyrm. Brave Habiteers, stand up and defeat this foul beast once and for all, but only if you believe you can stand against its immense power. <br><h3>Vice Part 1: </h3><br>How can you expect to fight the beast if it already has control over you? Don't fall victim to laziness and vice! Work hard to fight against the dragon's dark influence and dispel his hold on you!",
```

Comment:
@Blablux When paragraphs and headings are mixed, `<p>` tags should be used. 

This is better HTML:
`<p>... power. </p><h3>Vice Part 1: </h3><p>How ... </p>`

than this:
`... <br><h3>Vice Part 1: </h3><br>How ...`